pkgs/development/python-modules/ratarmount/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, isPy3k
+, indexed_bzip2
+, indexed_gzip
+, indexed_zstd
+, fusepy
+, lzmaffi
+, python-xz
+, rapidgzip
+, rarfile
+, ratarmountcore
+}:
+
+buildPythonPackage rec {
+  pname = "ratarmount";
+  version = "0.14.0";
+
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "3fea7487e2ae3acba73ec5db470c73021af55dc12a3234311de1e6036f7ea434";
+  };
+
+  propagatedBuildInputs = [ indexed_bzip2 indexed_gzip indexed_zstd fusepy lzmaffi python-xz rapidgzip rarfile ratarmountcore ];
```

Sorry for the back and forth

```suggestion
  propagatedBuildInputs = [ rarfile ratarmountcore ];
```

---

No problem. Thank you for the review!